lib/rules/selector-pseudo-class-no-unknown/index.js

```diff
@@ -68,26 +69,33 @@ const rule = (primary, secondaryOptions) => {
 						return;
 					}
 
-					// Ignore pseudo-elements
-					if (value.slice(0, 2) === '::') {
+					if (value.startsWith('::')) {
+						return;
+					}
+
+					const name = value.replace(/^:*/, '').toLowerCase();
+
+					if (levelOneAndTwoPseudoElements.has(name)) {
 						return;
 					}
 
 					if (optionsMatches(secondaryOptions, 'ignorePseudoClasses', pseudoNode.value.slice(1))) {
```

Comment:
It cannot be replaced by `name` because of the lowercasing.

Reply:
[suggest] Ah, I missed it. How about the following?

```js
const name = value.replace(/^:*/, '');
const lowerName = value.toLowerCase();

// ...

if (optionsMatches(secondaryOptions, 'ignorePseudoClasses', name)) {
```

Reply:
I think it's fine as is, I prefer not having to drill to find the blame for this line.
The less diff the better.

Reply:
Okay, leave it as-is. 👍🏼